### ISSUE TITLE
Sometimes takes multiple key presses to close bags

### ISSUE THREAD

**Issue by 2ndHouse** [tullamods/Bagnon#552]
Using Bagnon version 7.2.3. I can't figure out what causes it exactly, but sometimes I have to hit my toggle backpack (the blizzard keybind) 2-3 times to get the bags to close. Sometimes it closes correctly on the first key press, but not always. It's as if there is a lag with it at times. Never had this problem in 7.1 and earlier versions of Bagnon.

**Comment by SirThrocken** [comment]
Having the same issue trying to open/close bags myself, and only since the second to last update (using version 7.2.3 now); thought it was my keyboard for a while, but a stress test on the "B" key shows it's working properly.

**Comment by 2ndHouse** [comment]
Since we are both using Pawn as well I wonder if this is being caused by the Pawn issue in ticket #547

**Comment by otto-stierlitz** [comment]
I can confirm the issue without Pawn however: often multiple keypresses are needed to open and close bags.

**Comment by Jaliborc** [comment]
Do you have "Show Lua Errors" enabled at the interface options?

**Comment by 2ndHouse** [comment]
I don't think "Show Lua Errors" is even an option anymore. One of those Blizz removed I think. I am using Bugsack though, and I don't see any errors at all with Bagnon.

**Comment by 2ndHouse** [comment]
I can't be sure, as it's hard to pin down when it happens, but it seems like it might happen after visiting certain vendors, banks, AH, etc.

**Comment by 2ndHouse** [comment]
Just opened mail and had lots of items. Had to press the button 5-6 times to get bags to close. Maybe this is some kind of lag, and the more new items in your bag there is, the worse the lag?

**Comment by Lawmuse** [comment]
I'm have the same problem, intermittent multiple key strokes to close, sometime open, bagnon.  No lua errors are thrown. I created a new toon and disabled everything except bagnon and still noticed the problem from time to time. I tried changing the keybind to something else and everytime the binding is changed, it takes 5 to 7 tries to close bagnon once it opens.  Did this repeatedly with every change in keys.  after it closes, keys seem to work normally for a while.
